### DISCUSSION TITLE
Integrate Standalone with HA?

### DISCUSSION THREAD

**Discussion by jsamoore** [fl4p/batmon-ha#148] [Q&A]
TL:DR Can the stand alone Batmon-ha integrate with a Homeseer instance on another machine?

I apologize for my lack of knowledge as im new to Home Assistant and MQTT.  I have a separate barn with batteries that I wanted to monitor.  The standalone functionality seemed like the ticket to integrate with the HA server I have in the house.  I've got the standalone up and running on a Raspberry Pi 4 and have successfully connected to the BMS's .  I can also see that there is a successful connection to MQTT on the HA server.  I've installed the Add on, but then realized in looking at the logs that it is setup for a local instance of bluetooth.   Is this even possible?  If so, can someone point me in the right direction?  I don't want to waste my time if I'm fighting a losing battle.   Thank you.

**Comment by fl4p** [comment]
Just make sure your MQTT broker is reachable in your network. Then it doesn't matter where batmon is running, just point it to the same mqtt broker host.

**Reply by fl4p** [reply]
https://github.com/fl4p/batmon-ha/blob/master/doc/Standalone.md
